notes/Module-2/2.3.1-Hashing_Functions.md

```diff
@@ -0,0 +1,12 @@
+# Coach Notes
+
+## Coach comments
+Please add comments below which will help future coaches delivering this topic or suggestions for improvement of the content.
+
+|**Coach name**|**Delivery date**|**Comments**|
+|--------------|-----------------|------------|
+|Mandy|13/05/21|Hashing as a topic is great but the assignment was too difficult for my apprentices - maybe have as an extension task? Instead, I created [some slides](https://docs.google.com/presentation/d/1rol2aj8sEfGUrvCKCW4VovpZH56_D7-sbDJ7b9gzbIU/edit#slide=id.gd9c17e1e99_0_17) and gave them an assignment to create a password cracker of MD5/SHA non-salted passwords. This worked well as it instilled the importance of choosing strong passwords and demonstrated why salts and stronger algorithms like bcrypt should be used. We should ensure we discuss which hashing algorithms are considered secure & which are not (mentioned in my slides) in the notes|
```

Comment:
Too difficult even with the starter code? https://multiverselearningproducts.github.io/swe/notes/mod1/wk1/day4.html Maybe an extension activity then. So your slideshow is folded into the 2.0. Mod 2?